src/main/java/pl/ergohestia/ehj1/ivesta/services/DriverConverter.java

```diff
@@ -0,0 +1,29 @@
+package pl.ergohestia.ehj1.ivesta.services;
+
+import org.apache.commons.csv.CSVRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.ergohestia.ehj1.ivesta.model.Driver;
+
+public class DriverConverter {
+
+    private static final Logger SYSOUT = LoggerFactory.getLogger("SYSOUT");
```

Comment:
tutaj raczej klasyczny logger, po klasie. `"SYSOUT"` to konfiguracja loggera do interfejsu użytkownika